docs/architecture/how/resharding.md

```diff
@@ -47,61 +66,85 @@ StateSync in this phase would send the ``StateSplitRequest`` to the ``SyncJobsAc
+* near_resharding_batch_size and near_resharding_batch_count - those two metrics show how much data has been resharded. Both should be gradually increasing while the near_resharding_status. 
+  * While in the Scheduled state both metrics should remain 0. 
+  * While in the Building state both metrics should be gradually increasing. 
+  * While in the Finished state both metrics should remain at the same value. 
+* near_resharding_batch_prepare_time_bucket, near_resharding_batch_apply_time_bucket and near_resharding_batch_commit_time_bucket - those three metrics can be used to track the performance of resharding and fine tune throttling if needed. As a rule of thumb the combined time of prepare, apply and commit for a batch should remain at the 100ms-200ms level on average. Higher batch processing time may lead to disruptions in block processing, missing chunks and blocks. 
 
-## What should we improve to achieve 'stressless' resharding?
 
-Currently trie is split sequentially (shard by shard), and also sequentially within a shard - by iterating over all the elements. 
+### Throttling
 
-This process must finish within a single epoch - which is might be challenging (especially for larger archival nodes).
+The resharding process can be quite resource intensive and affect the regular operation of a node. In order to mitigate that as well as limit any need for increasing hardware specifications of the nodes throttling was added. Throttling slows down resharding to not have it impact other node operations. Throttling can be configured by adjusting the state_split_config in the node config file. 
 
-### Each shard should be handled independently
-This is a low-hanging fruit - but each shard's splitting should be done in a separate thread.
+* batch_size - controls the size of batches in which resharding moves data around. Setting a smaller batch size will slow down the resharding process and make it less resource consuming.
+* batch_delay - controls the delay between processing of batches. Setting a smaller batch delay will speed up the resharding process and make it more resource consuming. 
 
+The remainig fields in the StateSplitConfig are only intended for testing purposes and should remain set to their default values. 
 
-### Do trie splitting on the level of accounts, not key/values
+The default configuration for StateSplitConfig should provide a good and safe setting for resharding in the production networks. There is no need for node operators to make any changes to it unless they observe issues. 
 
-Most of the keys in our tries, are in the form of ``${some short prefix}${account_id}${long suffix}``, we could use this fact, and move the whole 'trie subtrees' around.
+A node needs to be restarted for the new config to take effect. This should be done only when absolutely necessary as restarting during resharding will interrupt it and resharding will need to start from beginning. 
```

Comment:
Have you considered making these config params updatable through SIGHUP? https://github.com/near/nearcore/blob/dd80b4c9ab4e399338231bf1c674b0217d31f2b7/neard/src/cli.rs#L570